lib/active_merchant/billing/gateways/world_net.rb

```diff
@@ -165,6 +184,22 @@ def cardholdername(payment)
         [payment.first_name, payment.last_name].join(' ').slice(0, 60)
       end
 
+      def add_new_stored_subscription(post, options)
+        sub_info = {}
+        sub_info[:merchantref] = options[:order_id]
+        sub_info[:name] = options[:subscription_name]
+        sub_info[:description] = options[:subscription_description]
+        sub_info[:periodtype] = options[:period_type]
+        sub_info[:length] = options[:length]
+        sub_info[:currency] = options[:currency]
+        sub_info[:recurringamount] = amount(options[:recurring_amount]) unless options[:type] && options[:type] == 'MANUAL'
```

Comment:
This seems tautological, just checking if it's `MANUAL` should be enough?

Reply:
Yes, now that I think about it, you're right ...